src/SIGV4ConnectionConfig.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { DataSourcePluginOptionsEditorProps, DataSourceSettings } from '@grafana/data';
+import { ConnectionConfig, ConnectionConfigProps } from './ConnectionConfig';
+
+import { AwsAuthDataSourceSecureJsonData, AwsAuthDataSourceJsonData } from './types';
+
+export const SIGV4ConnectionConfig: React.FC<DataSourcePluginOptionsEditorProps<any, any>> = (
```

Comment:
😭 no way to type this i guess right? Would unknown be better?

Reply:
Unfortunately not since HttpSettingsProps are types as any in core grafana. :( See https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/DataSourceSettings/types.ts#L22